promscale/installation/debian-ubuntu.md

```diff
@@ -0,0 +1,33 @@
+import PromscaleConnector from 'versionContent/_partials/_promscale-connector.mdx';
+
+# Install Promscale on Debian or Ubuntu
+You can install Promscale on any Debian or Ubuntu system where you have an
+already installed and working Prometheus or OpenTelemetry environment.
+
+## Install TimescaleDB and the Promscale extension
+To use Promscale you need a TimescaleDB database, as well as the Promscale
+extension. The Promscale extension contains support functions to improve
+the performance of Promscale.
+
+<procedure>
+
+### Installing TimescaleDB and Promscale extension
+
+1.  Install TimescaleDB following the instructions in the
+    [TimescaleDB install page][tsdb-install-self-hosted]
+1.  Install the Promscale extension:
+    ```
+    apt install promscale-extension-postgresql-14
```

Comment:
@VineethReddy02 I think we also provide packages for pg 12 and 13 right? Let's change this so it's clear.

I guess we could add a note saying something like replace 14 with the corresponding PostgreSQL major version. Versions 12, 13 and 14 are supported.

Or we change the command to apt install promscale-extension-postgresql-<pg_version> and explain they have to replace <pg_version> with 12, 13 or 14. I would use the former since I expect most people installing from scratch would be using 14and it reduces a little bit the amount of work the user has to do (they can just copy and paste)

Reply:
For the time being, I will add a note below this cmd. In the future, I will create a page for the compatibility matrix.